src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.java

```diff
@@ -3,12 +3,9 @@
 import javax.inject.Inject;
 
 import javafx.fxml.FXML;
-import javafx.scene.control.ButtonType;
```

Comment:
Please check your IDE setup. It seems that you did not import the JabRef code style. --> https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace

Reply:
That's true. Thanks @koppor!